src/source/feed-details.rst

```diff
@@ -602,11 +607,14 @@ Also, animated GIFs and PNGs are not supported.
 attributes
 """"""""""
 
-Use **attributes** field to provide additional information on your product in a structured way.
+Use **attributes** field to provide additional information on your item in a structured way,
+by providing a list of item :ref:`user_defined_attributes` that can be used to influence the ad relevance.
+**attributes** are category dependent.
```

Comment:
yup, referencing them is a good idea. I would just remove the "attributes are category dependent" then

Reply:
Here's an alternative:

Use the attributes field to provide additional information on your item in a structured way, by providing a list of [[attributes](https://ecg-icas.github.io/icas/doc/prod/attributes.html#attributes)](https://ecg-icas.github.io/icas/doc/prod/attributes.html#attributes). This can be used to influence the ad relevance and discoverability.

Attributes can be either *category* or *user-defined*, both sharing the same structure. User-defined attributes that match a category's predefined key and value are automatically recognized as category attributes.

Reply:
Also, please match the attributes in the TSV with the XML and maybe make it clear that the third TSV example is how you specify values for a "list" type of attribute